kotsadm/web/src/components/apps/AppDetailPage.jsx

```diff
@@ -129,14 +140,14 @@ class AppDetailPage extends Component {
   }
 
   getApp = async (slug = this.props.match.params.slug) => {
```

Comment:
This seems odd, especially since we're just checking for `this.props.match.params.slug` everywhere else in this function? does a slog get passed to this at all?

Reply:
it is used: https://github.com/replicatedhq/kots/blob/90426f19b1405d3cf6dec4430d6c96b9e2aae6b5/kotsadm/web/src/components/apps/AppDetailPage.jsx#L196

which is used in did mount and did update:
https://github.com/replicatedhq/kots/blob/90426f19b1405d3cf6dec4430d6c96b9e2aae6b5/kotsadm/web/src/components/apps/AppDetailPage.jsx#L205

Reply:
So I still am not really clear why were setting it to this.props.match.slug as a default value. Do we call it somewhere without passing a slug so we just set it to what're that param is?

Reply:
correct, there are few places where we're calling it without slug.

Reply:
I see. Ok sounds good